app/src/debug/java/com/artemzin/qualitymatters/developer_settings/DeveloperSettingsModule.java

```diff
@@ -68,4 +69,10 @@ public DeveloperSettingsPresenter provideDeveloperSettingsPresenter(@NonNull Dev
                                                                         @NonNull AnalyticsModel analyticsModel) {
         return new DeveloperSettingsPresenter(developerSettingsModelImpl, analyticsModel);
     }
+
+    @NonNull
+    @Provides
```

Comment:
Maybe add @Singleton so we'd reuse the Config object. @artem-zinnatullin what do you think about that?